pkg/services/authz/zanzana/store/migration/migrator.go

```diff
@@ -51,58 +75,19 @@ type migration struct {
 }
 
 func getMigrations(fs embed.FS, path string) ([]migration, error) {
-	entries, err := fs.ReadDir(path)
-	if err != nil {
-		return nil, fmt.Errorf("failed to read migration dir: %w", err)
-	}
-
-	// parseStatements extracts statements from a sql file so we can execute
-	// them as separate migrations. OpenFGA uses Goose as their migration egine
-	// and Goose uses a single sql file for both up and down migrations.
-	// Grafana only supports up migration so we strip out the down migration
-	// and parse each individual statement
-	parseStatements := func(data []byte) ([]string, error) {
-		scripts := strings.Split(strings.TrimPrefix(string(data), "-- +goose Up"), "-- +goose Down")
-		if len(scripts) != 2 {
-			return nil, errors.New("malformed migration file")
-		}
-
-		// We assume that up migrations are always before down migrations
-		parts := strings.SplitAfter(scripts[0], ";")
-		stmts := make([]string, 0, len(parts))
-		for _, p := range parts {
-			p = strings.TrimSpace(p)
-			if p != "" {
-				stmts = append(stmts, p)
-			}
-		}
+	migrations := make([]migration, 0, 1)
```

Comment:
Once you have only one migration, you can init array straight with it, no need to append